NDTensors/src/fill.jl

```diff
@@ -0,0 +1,64 @@
+# This is a file to write generic fills for NDTensors.
+#  This includes random fills, zeros, ...
+
+function generic_randn(
+  StoreT::Type{<:Dense{ElT,DataT}}, dim::Integer=0
+) where {DataT<:AbstractArray{ElT}} where {ElT}
+  data = generic_randn(DataT, dim)
+  return StoreT(data)
+end
+
+function generic_randn(StoreT::Type{<:Dense}, dim::Integer=0)
```

Comment:
Isn't this function the same as the one above?

Reply:
Oops forgot that this is used to distinguish `Dense` when the eltype/datatype are specified or not specified.